dockerfiles/c-23.Dockerfile

```
@@ -0,0 +1,38 @@
+ENV PATH="${CMAKE_BIN}:$PATH"
+
+RUN git clone https://github.com/microsoft/vcpkg.git && \
+    ./vcpkg/bootstrap-vcpkg.sh -disableMetrics
+
+ENV VCPKG_ROOT="/vcpkg"
+ENV PATH="${VCPKG_ROOT}:$PATH"
+
+WORKDIR /app
+
+# .git & README.md are unique per-repository. We ignore them on first copy to prevent cache misses
+COPY --exclude=.git --exclude=README.md . /app
```

Hadolint (2.12.0)

[error] invalid flag: --exclude

(DL1000)